Dockerfile

```diff
@@ -37,7 +40,7 @@ RUN dnf -y module enable ruby:2.7 nodejs:16 mysql:8.0 \
 # https://issues.redhat.com/browse/RHEL-14497
 RUN mkdir /tmp/memkind \
     && cd /tmp/memkind \
-    && dnf download --source memkind \
+    && dnf download http://mirror.stream.centos.org/9-stream/AppStream/source/tree/Packages/memkind-1.11.0-2.el9.src.rpm \
```

Comment:
why is this change needed :thinking:

Reply:
I think this was solved in https://github.com/3scale/porta/pull/3847, so it shouldn't be needed here neither.

Reply:
Yeah, for some reason this command didn't work for me for some time.... Then magically it started working :upside_down_face: